docs/MigrationGuide.md

```diff
@@ -16,6 +16,8 @@ To see the biggest differences please consult the [changelog](Changelog.md).
   * Now method `returnType` from `EagerServiceWithStaticParameters` requires implicit nodeId param
 * [#6437](https://github.com/TouK/nussknacker/pull/6437) Removed deprecated operation to create a scenario:
   POST `/api/processes/{name}/{category}`. POST `/api/processes` should be used instead.
+* [#6213](https://github.com/TouK/nussknacker/pull/6213) Improvement: Load resource config only in test context
```

Comment:
`WithConfig` from test-utils modules behaviour changes: it only parse given config, without resolving reference configs, system variables etc.